src/components/ComponentDemo/ComponentDemo.js

```diff
@@ -162,12 +165,18 @@ const ComponentDemo = ({ children, src, scope, noInline, components }) => {
               )}
 
               <Code links={links} code={code} src={src} className={codeRow}>
-                <LiveEditor
-                  padding={16}
-                  style={{ overflowX: 'auto', whiteSpace: 'pre' }}
-                  onChange={(updatedCode) => setCode(updatedCode)}
-                  className={editorContainer}
-                />
+                <label htmlFor="live-editor-label">
+                  <span className="live-editor-label">
+                    `Live editor for the ${components[0].label} component`
+                  </span>
+                  <LiveEditor
+                    padding={16}
+                    style={{ overflowX: 'auto', whiteSpace: 'pre' }}
+                    onChange={(updatedCode) => setCode(updatedCode)}
+                    className={editorContainer}
+                    textareaId="live-editor-label"
+                  />
+                </label>
```

Comment:
```suggestion
                <label htmlFor="live-editor-label">
                    <span className="live-editor-label">{labelText}</span>
                  </label>
                  <LiveEditor
                    padding={16}
                    style={{ overflowX: 'auto', whiteSpace: 'pre' }}
                    onChange={(updatedCode) => setCode(updatedCode)}
                    className={editorContainer}
                    textareaId="live-editor-label"
                  />
```

I'm still seeing a violation with this format. I don't think we need to wrap the entire `LiveEditor` in the `label`.

The label text was also being parsed strangely rendering it here for some reason. I moved it to before the render method and added something like

```jsx
const labelText = `Live editor for the ${components[0].label} component`;
```